src/nls/messages_pt.json

```
@@ -1,33 +1,33 @@
 {
-  "carbon.listBoxMenuIcon.close.menu": "",
-  "carbon.listBoxMenuIcon.open.menu": "",
-  "carbon.listBoxSelection.clear.all": "",
-  "carbon.listBoxSelection.clear.selection": "",
-  "carbon.table.all.collapse": "",
-  "carbon.table.all.expand": "",
+  "carbon.listBoxMenuIcon.close.menu": "Fechar menu",
+  "carbon.listBoxMenuIcon.open.menu": "Abrir menu",
+  "carbon.listBoxSelection.clear.all": "Limpar todos os itens selecionados",
+  "carbon.listBoxSelection.clear.selection": "Limpar item selecionado",
+  "carbon.table.all.collapse": "Retrair todas as linhas",
+  "carbon.table.all.expand": "Expandir todas as linhas",
   "carbon.table.all.select": "Selecionar todas as linhas",
   "carbon.table.all.unselect": "Desmarcar todas as linhas",
   "carbon.table.batch.cancel": "Cancelar",
   "carbon.table.batch.item.selected": "1 item selecionado",
   "carbon.table.batch.items.selected": "{totalSelected, plural, other {# itens}} selecionados",
-  "carbon.table.row.collapse": "",
-  "carbon.table.row.expand": "",
+  "carbon.table.row.collapse": "Retrair linha atual",
+  "carbon.table.row.expand": "Expandir linha atual",
   "carbon.table.row.select": "Selecionar linha",
   "carbon.table.row.unselect": "Desmarcar linha",
-  "dashboard.about.description": "",
-  "dashboard.about.documentation": "",
-  "dashboard.about.environmentDetails": "",
-  "dashboard.about.error": "",
+  "dashboard.about.description": "Tekton é um poderoso e flexível framework open-source para criação de sistemas CI/CD, permitindo que desenvolvedores possam construir, testar, e implantar através de provedores de nuvem e sistemas on-premises",
+  "dashboard.about.documentation": "Documentação e recursos",
+  "dashboard.about.environmentDetails": "Detalhes de ambiente",
+  "dashboard.about.error": "Erro ao receber dados",
   "dashboard.about.isReadOnly": "",
   "dashboard.about.logoutURL": "",
-  "dashboard.about.missingProperties": "",
-  "dashboard.about.title": "",
+  "dashboard.about.missingProperties": "Não foi possível encontrar: {errorsFound}",
+  "dashboard.about.title": "Sobre Tekton", 
```

```suggestion
  "dashboard.about.title": "Sobre Tekton",
```

Can you remove the trailing space at the end of this line? That should resolve the issue with the tests.